app/rpc/RPC.ts

```diff
@@ -784,7 +784,7 @@ export default class RPC {
     const info = await RPC.rpc_getInfoObject();
 
     if (info) {
-      await this.fnSetInfo(info);
+      this.fnSetInfo(info);
```

Comment:
Aha, I remember my language server complaining to me about unneeded awaits... Nice.

Reply:
Yes, the theory is if the function is returning `void`, you don't need `await` anymore.